### ISSUE TITLE
Node gives warning that there is no output after upgrading from 2.00.3 to 2.00.6

### ISSUE THREAD

**Issue by gjannece** [HaveAGitGat/Tdarr#391]
I'm getting "Tdarr ALERT: NO OUTPUT FILE PRODUCED:" after upgrading to 2.00.6. 

Permissions are wide open and still getting the issues.

`{
  _id: '/media/NAS/TV-Shows/Video/Video - S01E10 - Episode 10 WEBDL-1080p.mkv',
  file: '/media/NAS/TV-Shows/Video/Video - S01E10 - Episode 10 WEBDL-1080p.mkv',
  container: 'mkv',
  ffProbeRead: 'success',
  createdAt: 1614634142388,
  lastPluginDetails: 'none',
  bit_rate: 5625793.885108086,
  statSync: {
    dev: 54,
    mode: 33279,
    nlink: 1,
    uid: 1001,
    gid: 1001,
    rdev: 0,
    blksize: 131072,
    ino: 47873,
    size: 2136091435,
    blocks: 4172056,
    atimeMs: 1614634134112.6465,
    mtimeMs: 1527275853000,
    ctimeMs: 1614632415586.994,
    birthtimeMs: 0,
    atime: '2021-03-01T21:28:54.113Z',
    mtime: '2018-05-25T19:17:33.000Z',
    ctime: '2021-03-01T21:00:15.587Z',
    birthtime: '1970-01-01T00:00:00.000Z'
  },
  file_size: 2037.1355390548706,
  ffProbeData: { streams: [ [Object], [Object], [Object] ] },
  meta: {
    SourceFile: '/media/NAS/TV-Shows/Video/Video - S01E10 - Episode 10 WEBDL-1080p.mkv',
    errors: [],
    Duration: 3037.568,
    DefaultDuration: 0.032,
    ExifToolVersion: 12.1,
    FileName: 'Video - S01E10 - Episode 10 WEBDL-1080p.mkv',
    Directory: '/media/NAS/TV-Shows/Video',
    FileSize: '2037 MB',
    FileModifyDate: {
      year: 2018,
      month: 5,
      day: 25,
      hour: 12,
      minute: 17,
      second: 33,
      millisecond: 0,
      tzoffsetMinutes: -420,
      rawValue: '2018:05:25 12:17:33-07:00'
    },
    FileAccessDate: {
      year: 2021,
      month: 3,
      day: 1,
      hour: 13,
      minute: 28,
      second: 54,
      millisecond: 0,
      tzoffsetMinutes: -480,
      rawValue: '2021:03:01 13:28:54-08:00'
    },
    FileInodeChangeDate: {
      year: 2021,
      month: 3,
      day: 1,
      hour: 13,
      minute: 0,
      second: 15,
      millisecond: 0,
      tzoffsetMinutes: -480,
      rawValue: '2021:03:01 13:00:15-08:00'
    },
    FilePermissions: 'rwxrwxrwx',
    FileType: 'MKV',
    FileTypeExtension: 'mkv',
    MIMEType: 'video/x-matroska',
    EBMLVersion: 1,
    EBMLReadVersion: 1,
    DocType: 'matroska',
    DocTypeVersion: 4,
    DocTypeReadVersion: 2,
    TimecodeScale: '1 ms',
    MuxingApp: 'libebml v1.3.4 + libmatroska v1.4.5',
    WritingApp: "mkvmerge v9.7.1 ('Pandemonium') 64bit",
    DateTimeOriginal: {
      year: 2017,
      month: 9,
      day: 3,
      hour: 17,
      minute: 35,
      second: 32,
      millisecond: 0,
      tzoffsetMinutes: 0,
      rawValue: '2017:09:03 17:35:32Z'
    },
    VideoCodecID: 'V_MPEG4/ISO/AVC',
    VideoFrameRate: 23.976,
    ImageWidth: 1920,
    ImageHeight: 1080,
    DisplayWidth: 1920,
    DisplayHeight: 1080,
    AudioCodecID: 'A_AC3',
    AudioSampleRate: 48000,
    AudioChannels: 6,
    TrackNumber: 3,
    TrackType: 'Subtitle',
    TrackDefault: 'No',
    CodecID: 'S_TEXT/UTF8',
    ImageSize: '1920x1080',
    Megapixels: 2.1
  },
  mediaInfo: { '@ref': '', track: [ [Object], [Object], [Object], [Object] ] },
  hasClosedCaptions: false,
  DB: 'SIEFfPFb1',
  bumped: false,
  HealthCheck: '',
  TranscodeDecisionMaker: '',
  processingStatus: false,
  fileMedium: 'video',
  infoLog: '',
  video_codec_name: 'h264',
  audio_codec_name: '',
  video_resolution: '1080p',
  lastHealthCheckDate: 0,
  lastTranscodeDate: 0,
  history: '',
  oldSize: 0,
  newSize: 0,
  tPosition: 0,
  hPosition: 0
}
{
  processFile: true,
  preset: ',-map 0 -c:v libx265 -b:v 2682k -minrate 1877k -maxrate 3486k -bufsize 5365k -c:a copy -c:s copy -max_muxing_queue_size 9999 ',
  handBrakeMode: false,
  FFmpegMode: true,
  reQueueAfter: true,
  infoLog: 'Container for output selected as mkv. \n' +
    'Current bitrate = 5365 \n' +
    'Bitrate settings: \n' +
    'Target = 2682 \n' +
    'Minimum = 1877 \n' +
    'Maximum = 3486 \n' +
    'File is not hevc or vp9. Transcoding. \n',
  container: '.mkv',
  lastPluginDetails: {
    source: 'Community',
    id: 'Tdarr_Plugin_MC93_Migz1FFMPEG_CPU',
    number: '2/3'
  },
  ffmpegMode: true
}
[2021-03-01T21:29:05.079] [INFO] Tdarr_Node - verbose:Sub-worker exit status received
[2021-03-01T21:29:05.080] [WARN] Tdarr_Node - Tdarr ALERT: NO OUTPUT FILE PRODUCED:
/media/NAS/Tdarr-cache/Videofile - S01E10 - Episode 10 WEBDL-1080p-TdarrCacheFile-7W8XkTX6E.mkv
[2021-03-01T21:29:05.085] [INFO] Tdarr_Node - verbose:Sub-worker exited
[2021-03-01T21:29:09.504] [INFO] Tdarr_Node - verbose:Requesting item
[2021-03-01T21:29:09.504] [INFO] Tdarr_Node - verbose:Item request received
[2021-03-01T21:29:09.504] [INFO] Tdarr_Node - verbose:Getting new task`

**Comment by HaveAGitGat** [comment]
Hi thanks for opening an issue. Please fill out the requested information (including configs and logs) when creating a bug report, thanks:

![image](https://user-images.githubusercontent.com/43864057/109926465-4602fd00-7cc3-11eb-99a0-a260da4494db.png)

**Comment by HaveAGitGat** [comment]
Feel free to reopen if you have the logs. Worth trying 2.00.07 as it might fix your issue.
